app/Domain/Goalcanvas/Templates/bigRockDialog.blade.php

```diff
@@ -25,10 +25,11 @@
 <script>
     jQuery(document).ready(function(){
 
-        @if (isset($_GET['closeModal']))
+        <?php if (isset($_GET['closeModal'])): ?>
```

Comment:
Why not `@if` ? This should be blade

Reply:
I was getting syntax error in the editor, while using `@if` inside the script tag, might be vs setting issue